packages/typescript-estree/src/ts-estree/ts-estree.ts

```diff
@@ -115,6 +115,7 @@ export interface LineComment extends BaseToken {
 export type Comment = BlockComment | LineComment;
 export type Token =
   | BooleanToken
+  | Comment
```

Comment:
Seems this was left out of the union - might have been on purpose but can't find any reference to it.

[ESLint passes these nodes to `fixer` methods](https://github.com/eslint/eslint/blob/master/lib/rules/multiline-comment-style.js#L393), so I think it's fine.

Happy to break this out into another PR if you'd like to land it for the next release without having to review everything else :)